src/backend/bs_srcserver

```diff
@@ -8428,6 +8428,8 @@ if ($BSConfig::workersrcserver) {
   $conf->{'port2'} = $wport if $wport != $port;
 }
 
+# always create the event directory if it not exist
+BSUtil::mkdir_p_chown("$eventdir", $BSConfig::bsuser, $BSConfig::bsgroup) unless -d "$eventdir";
```

Comment:
@mlschroe I`m wondering if we simply should backport ff28d0353b - $eventdir should be created by function "addevent"